src/sass/Abstracts/_Variables.scss

```
@@ -0,0 +1 @@
+// Variables: assign a value to a name for reusability and readability
```

```suggestion
// Variables: assign a value to a name for reusability and readability.
```

Nit: missing period.